packages/sdk/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@module-federation/sdk",
-  "version": "1.0.0-canary.1",
+  "version": "1.0.0-canary.3",
```

Comment:
@ScriptedAlchemy   not sure the version can set this , but i see the latest version is  `1.0.0-canary.3` 
![image](https://github.com/module-federation/universe/assets/41466093/8c1f79f5-cca4-4e0d-a7e0-fe1c194d7ceb)